libs/shared/ui/src/lib/styles/main.scss

```diff
@@ -26,6 +26,7 @@ body {
   overflow-x: hidden;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  @apply bg-element-light-lighter-400;
```

Comment:
Wow we missed that the body of the website was not white ??

Reply:
Why not using the new way we found for tailwind apply ? 
`background-color: theme(blablalb)`

Reply:
Oh yes ! I change that :)